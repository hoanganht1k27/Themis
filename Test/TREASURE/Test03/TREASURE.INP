16 16
501892013 866531122 696526688 230242036 906475194 668200251 588855230 634586856 828037552 51573131 914454267 317850596 623029077 606325926 340334112 181071231
3 4
9 1
8 16
6 7
7 8
10 4
6 16
15 11
14 15
16 6
14 15
14 2
8 1
12 16
3 7
3 6
2 4 15 7 11 6 1 3 12 9 5 14 16 10 8 13
