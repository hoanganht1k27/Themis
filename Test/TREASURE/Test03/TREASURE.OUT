6598443459
6598443459
6598443459
6546870328
6206536216
6025464985
5795222949
5293330936
4426799814
3820473888
3231618658
2913768062
2217241374
1549041123
634586856
0
